astro/src/content/docs/apis/_event-types.mdx

```diff
@@ -2,43 +2,44 @@ import EnterprisePlanBlurbApi from 'src/content/docs/_shared/_enterprise-plan-bl
-* <code>user.password.breach</code> - When Reactor detects a user is using a potentially breached password (requires an activated license)  <AvailableSince since="1.15.0"/> <br/>  <PremiumPlanBlurbApi>the ``user.password.breach`` event </PremiumPlanBlurbApi>
-* <code>user.password.reset.send</code> - When a forgot password email has been sent to a user  <AvailableSince since="1.30.0"inline={true}/> <br/>  <EnterprisePlanBlurbApi>the ``user.password.reset.send`` event</EnterprisePlanBlurbApi>
-* <code>user.password.reset.start</code> - When the process to reset a user password has started  <AvailableSince since="1.30.0"/> <br/>  <EnterprisePlanBlurbApi>the ``user.password.reset.start`` event</EnterprisePlanBlurbApi>
-* <code>user.password.reset.success</code> -  When a user has successfully reset their password  <AvailableSince since="1.30.0"/> <br/>  <EnterprisePlanBlurbApi>the ``user.password.reset.success`` event</EnterprisePlanBlurbApi>
-* <code>user.password.update</code> - When a user has updated their password  <AvailableSince since="1.30.0"/> <br/>  <EnterprisePlanBlurbApi>the ``user.password.update`` event</EnterprisePlanBlurbApi>
-* <code>user.reactivate</code> - When a user is reactivated
-* <code>user.registration.create</code> - When a user registration is created  <AvailableSince since="1.6.0"/>
-* <code>user.registration.create.complete</code> - When a user registration create transaction has completed  <AvailableSince since="1.30.0"/>
-* <code>user.registration.delete</code> - When a user registration is deleted  <AvailableSince since="1.6.0"/>
-* <code>user.registration.delete.complete</code> - When a user registration delete transaction has completed  <AvailableSince since="1.30.0"/>
-* <code>user.registration.update</code> - When a user registration is updated  <AvailableSince since="1.6.0"/>
-* <code>user.registration.update.complete</code> -  When a user registration update transaction has completed  <AvailableSince since="1.30.0"/>
-* <code>user.registration.verified</code> - When a user completes registration verification  <AvailableSince since="1.8.0"/>
-* <code>user.two-factor.method.add</code> - When a user has added a two-factor method  <AvailableSince since="1.30.0"/> <br/>  <EnterprisePlanBlurbApi>the ``user.two-factor.method.add`` event</EnterprisePlanBlurbApi>
-* <code>user.two-factor.method.remove</code> - When a user has removed a two-factor method  <AvailableSince since="1.30.0" /> <br/>  <EnterprisePlanBlurbApi>the ``user.two-factor.method.remove`` event</EnterprisePlanBlurbApi>
-* <code>user.update</code> - When a user is updated
-* <code>user.update.complete</code> - When a user update transaction has completed  <AvailableSince since="1.30.0"/>
+* `audit-log.create` - When an audit log is created <AvailableSince since="1.30.0" />
+* `event-log.create` - When an event log is created  <AvailableSince since="1.30.0" />
+* `identity.verified` - When a user's identity is verified <AvailableSince since="1.99.9" />
```

Comment:
We don't have a `user.` prefix on our event. should we?

Reply:
Yup. In a pre-verified situation the event would still be sent.

Reply:
So would the expectation be that this event would be immediately followed by `user.create`?

Reply:
> So would the expectation be that this event would be immediately followed by user.create?

Shortly followed, yes.

> I think this should be user.identity.verified unless we think an identity would ever be something we managed completely independently of a user (and I think pre-verification is still in the context of a user, just one that doesn't exist yet)


I think I favor renaming it too, but was checking.

Reply:
captured the rename in ENG-2047.